README.md

```diff
@@ -17,9 +16,10 @@ Inputs:
 `token`|**Required** token to authorize calls to GitHub API, can be ${{github.token}} to create a deployment for the same repo
 `type`|**Required** type of an action. Should be `create` to create a deployment
 `logs`|url to the deployment logs
-`environment`|environment to create a deployments in, default to `context.ref` without prefixes (`'refs/heads/'`, `'deploy-'`), i.e. branch name
+`environment`|environment to create a deployments in, default to `$context.ref` without prefixes (`'refs/heads/'`, `'deploy-'`), i.e. branch name
 `environment_url`|link to the deployed application
 `description`|optional description, defaults to `"deployed by $context.actor"`
+`job_status`| pass `${{job.status}}` to set the deployment completion status post script accordingly
```

Comment:
@gordey4doronin @lumaxis that's a dirty hack to track deployment status in post script, unless we find a way to query ${{job.status}} in JS code.